src/objects/Arrow.gd

```diff
@@ -0,0 +1,20 @@
+extends KinematicBody
+
+
+const GRAVITY = 9.8
+
+var archer
```

Comment:
#pedantry - `var archer: Node`, maybe even `var archer: KinematicBody` but I don't want it to be too rigid just in case we change something